messaging/src/main/java/io/lantern/messaging/Messaging.kt

```diff
@@ -850,6 +876,67 @@ class Messaging(
         }
     }
 
+    @Throws(IllegalArgumentException::class)
+    fun introduce(vararg recipientIds: String) {
```

Comment:
As per https://kotlinlang.org/docs/functions.html#variable-number-of-arguments-varargs `recipientIds` can be an array of String ids like `[ 'contactId.id1', 'contactId.id2', ...]` right?

Reply:
Yes, you just need to use the [spread](https://www.baeldung.com/kotlin/varargs-spread-operator#spread-operator) operator on the array in order to pass it to the vararg function.

Reply:
So for example:

```kotlin
val contacts = listOf('id1', 'id2', 'id3')
messaging.introduce(*contacts)
```

Reply:
This might be difficult/ugly to call from Java code though, so let me chance the function parameter to a list. One sec.

Reply:
Okay, I updated the signature to take a List.

Reply:
Thanks - however I can't see the change? 🤔 Did the commit get pushed correctly?

Reply:
It's in GitHub ...